src/pudl/settings.py

```diff
@@ -243,41 +244,84 @@ class Eia860Settings(GenericDatasetSettings):
         data_source: DataSource metadata object
         years: list of years to validate.
         eia860m: whether or not to incorporate an EIA-860m month.
-        eia860m_year_month ClassVar[str]: The 860m year-month to incorporate.
+        eia860m_year_months ClassVar[str]: The 860m year-months to incorporate.
     """
 
     data_source: ClassVar[DataSource] = DataSource.from_id("eia860")
     years: list[int] = data_source.working_partitions["years"]
+    eia860m: bool = Field(validate_default=True, default=True)
+    all_eia860m_year_months: list[str] = DataSource.from_id(
+        "eia860m"
+    ).working_partitions["year_months"]
+    eia860m_year_months: list[str] = Field(validate_default=True, default=[])
```

Comment:
I made this default value empty again because it felt weird to have a non-empty default option for `eia860_year_months` when you could specify `eia860m = False`.

Reply:
So the default is `[]` but if `eia860m=True` and the default value is used, the correct final validated value for `eia860m_year_months` will be dynamically determined by the validator below?

Reply:
Yes, and the `settings_test` `test_eia860m_after_eia860` function should verify that